city_scrapers/spiders/il_sports_facilities_authority.py

```diff
@@ -0,0 +1,105 @@
+from city_scrapers_core.constants import BOARD, COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class IlSportsFacilitiesAuthoritySpider(CityScrapersSpider):
+    name = "il_sports_facilities_authority"
+    agency = "Illinois Sports Facilities Authority"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.isfauthority.com/governance/board-meetings/"]
+    location = {
+        "name": "Authority offices",
+        "address": "Guaranteed Rate Field, 333 West 35th Street, Chicago, IL",
```

Comment:
Small thing, but for consistency and since we know it's static it would be good to include the ZIP code and update this to be:

```python
{
  "name": "Authority offices, Guaranteed Rate Field",
  "address": "333 W 35th St, Chicago, IL 60616",
}
```

Reply:
Fixed with commit edb717e, should now match the address you specified